src/alarms/notifications/NewNotification.js

```diff
@@ -413,10 +436,30 @@ class NewNotification extends Component {
       return false;
     }
   }
-  handleResetTimeseriesNestedAsset(timeseriesNestedAsset) {
-    this.handleSetTimeseriesNestedAsset("");
+  handleResetTimeseriesNestedAsset() {
+    this.setState({
+      timeseriesNestedAssets: [],
+      timeseriesNestedAsset: ""
+    });
+    // Also reset timeseries uuids.
+    this.handleResetTimeseriesUuid();
   }
   handleSetTimeseriesUuid(timeseriesUuid) {
+    fetchTimeseriesUuidsFromAsset(
+      this.state.timeseriesAssetType,
+      this.state.timeseriesAssetId
+    ).then(data => {
+      let uuids = data.map(function(timeserie) {
+        if (timeserie.uuid) {
+          return timeserie.uuid;
+        } else {
+          return timeserie;
+        }
```

Comment:
timeserie.uuid || timeserie, maar ik vind dat verwarrend: wat voor datatype is timeserie hier?